src/Commands/Connection/SetCommand.php

```diff
@@ -16,19 +16,19 @@ class SetCommand extends TerminusCommand implements SiteAwareInterface
     use SiteAwareTrait;
 
     /**
-     * Set git or sftp connection mode on a site's dev or multidev environment
+     * Sets Git or SFTP connection mode on a Dev or Multidev environment.
```

Comment:
Please, please don't capitalize `dev` and `multidev`. `dev` because if a user tries `terminus connection:set <site>.Dev <mode>`, it would err and "Multidev" because it irks me. 😆